statsmodels/tsa/varma_process.py

```diff
@@ -464,11 +466,14 @@ class VarmaPoly(object):
 
 
     '''
+
+    nvars = deprecated_alias('nvars', 'neqs', '0.11.0')
+
     def __init__(self, ar, ma=None):
         self.ar = ar
         self.ma = ma
         nlags, nvarall, nvars = ar.shape
```

Comment:
`nvars` -> `negs` here

Reply:
To keep the diff small, I only changed it in the one line that sets `self.nvars = nvars` to set `self.neqs = nvars` instead.